### PR TITLE
Update app to run in Tethys 4

### PR DESCRIPTION
**Pull request by EricJJones**
Co-authored-by: Nathan Swain <swainn@users.noreply.github.com>

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (swainn) [inline .vscode/settings.json]: Remove this file. Use this command to remove it from the repository, but keep it locally:

```
git rm --cached .vscode/settings.json
```

Then add it to the .gitignore file so it doesn't get committed again in the future.

File: tethysapp/data_rods_explorer/controllers.py [hunk 2]
```diff
@@ -122,6 +134,7 @@ def plot(request):
             varunit = get_wms_vars()[post['model']][post['variable']][2]
             point_lon_lat = post['pointLonLat']
             datarod_ts, datarods_urls_dict = get_data_rod_plot(post, point_lon_lat)
+            print('got this far')
```
Comment (swainn) [inline tethysapp/data_rods_explorer/controllers.py]: Remove debug print statement.

File: tethysapp/data_rods_explorer/public/css/main.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #map-loading {
     position: absolute;
-    z-index: 1000;
+    /* z-index: 1000; */
```
Comment (swainn) [inline tethysapp/data_rods_explorer/public/css/main.css]: Remove commented out code.

File: tethysapp/data_rods_explorer/public/js/load_pages.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ function onClickLink(link, navItem) {
     } else {
         $('.nav-link').removeClass('open');
         $(link).addClass('open');
-        $('.nav-item').addClass('hidden');
+        //$('.nav-item').addClass('hidden');
         $('#nav-' + navItem).removeClass('hidden');
```
Comment (swainn) [inline tethysapp/data_rods_explorer/public/js/load_pages.js]: Why is line 15 commented out? `'hidden'` probably should be changed to `'d-none'` here.

File: tethysapp/data_rods_explorer/public/js/cookies.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+var docCookies = {
```
Comment (swainn) [inline tethysapp/data_rods_explorer/public/js/cookies.js]: Delete this file. I'm pretty sure we replaced the reference to this file with one that comes with Tethys (`{{ tethys.doc_cookies.script_tag|safe }}`).

File: tethysapp/data_rods_explorer/model_objects.py [hunk 6]
```diff
@@ -253,26 +253,27 @@ def parse_fences_from_file():
...
-                wbound = linevals[3].split(', ')[3]
-                model_fences[linevals[0]] = {
-                    'start_date': start_date,
-                    'end_date': end_date,
-                    'extents': {
-                        'maxY': nbound,
-                        'maxX': ebound,
-                        'minY': sbound,
-                        'minX': wbound
+                if len(linevals) > 1:
```
Comment (swainn) [inline tethysapp/data_rods_explorer/model_objects.py]: Probably safer for this to be `if len(linevals) > 3:`, no?

**Review by swainn — CHANGES_REQUESTED** [review]
@EricJJones see comments.

File: tethysapp/data_rods_explorer/public/js/load_pages.js (hunk 4 above)
Comment (EricJJones) [reply tethysapp/data_rods_explorer/public/js/load_pages.js]: I don't recall commenting that out, nor see any reason it should be.  Uncommenting and changing to d-none

File: tethysapp/data_rods_explorer/model_objects.py (hunk 6 above)
Comment (EricJJones) [reply tethysapp/data_rods_explorer/model_objects.py]: Yep

**Review by swainn — APPROVED** [review]

**Review by swainn — APPROVED** [review]

**Comment by rileyhales** [comment]
@swainn @msouff Does this app even get used? I understood this was a CUAHSI favorite. @danames should decide what to do with this.

**Pull request merged**